rpc/rpc-wrappers.go

```diff
@@ -397,6 +398,40 @@ func NewValidationAuthorityServer(serverQueue string, channel *amqp.Channel, imp
 		return nil
 	})
 
+	rpc.Handle(MethodCheckCAARecords, func(req []byte) []byte {
+		var caaReq struct {
```

Comment:
This structure is redundant between this and CheckCAARecords below. Should be pulled out to a shared struct.